indico/modules/receipts/default_templates/attendance/template.html

```diff
@@ -0,0 +1,95 @@
+{% set personal_data = registration.personal_data %}
+{% set today = now_utc() %}
+
+<h1>Certificate of Attendance</h1>
+
+<aside id="title">
+  <div class="title">
+    <!-- Title -->
+    <h2>{{ event.title }}</h2>
+    <!-- Date(s) -->
+    {% if event.start_dt.date() != event.end_dt.date() %}
+      {{ format_interval(event.start_dt, event.end_dt, format='dMMMMy') }}
+    {% else %}
+      {{ event.start_dt | format_date('dd MMM YYYY') }}
+    {% endif %}
+
+    <!-- Venue -->
+    {% if custom_fields.venue %}
+      - {{ custom_fields.venue }}
+    {% elif event.venue_name %}
+      - {{ event.venue_name }}
+    {% endif %}
+
+    {% if custom_fields.add_url %}
+      <p>{{ event.url }}</p>
+    {% endif %}
+  </div>
+
+  <!-- Logo -->
+  {% if custom_fields.logo_url %}
+    <img src="{{ custom_fields.logo_url }}">
+  {% endif %}
+</aside>
+
+<aside id="addresses">
+  <!-- Address of organizer -->
+  <address id="from">
+    {{ custom_fields.address_from }}
+  </address>
+</aside>
+
+{%- if event.start_dt.date() != event.end_dt.date() %}
+  {% set date = 'between {} and {}'.format(event.start_dt | format_date('dd.MM.YYYY'), event.end_dt | format_date('dd.MM.YYYY')) %}
```

Comment:
OK, after seeing this again, I think we really need to provide multiple placeholders:

- `{date_range}` for the current text w/ the automated single/multi day logic + english text pieces
- `{start_date}` and `{end_date}` for the individual dates